frontend/src/libs/types/types.ts

```diff
@@ -1,9 +1,11 @@
 export { type AsyncThunkConfig } from './async-thunk-config.type.js';
 export {
   type FormControl,
+  type FormControllerRenderProps,
   type FormFieldErrors,
   type FormFieldPath,
   type FormFieldValues,
+  type FormPath,
 } from './form/form.type.js';
 export { type SidebarRoute } from './sidebar-route.type.js';
 export { type ValueOf } from './values-of.type.js';
```

Comment:
I think the file that only exports ValueOf should be named './value-of.type.js'
```suggestion
export { type ValueOf } from './value-of.type.js';
```

Reply:
Yeah, done. Nevertheless this change should have been in sidebar wrapper pr, but at least here will be correct